### PR TITLE
merge the publication-specific bits into index.qmd

### PR DESCRIPTION
**Pull request by floe**
I've merged the PDF-specific parts into `index.qmd`, only visible difference on the website is that we'll also have a reference section now.

### CODE REVIEW

**Comment by floe** [comment]
P.S. Figured out how to hide the references for the online version, now they only show up in the PDF.

File: index.qmd [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+    myrefs: "References"
+    author:
+      - "[Lonni Besançon](https://lonnibesancon.me/), Linköping University"
+      - "[Florian Echtler](https://floe.butterbrot.org/), Aalborg University"
+      - "[Matthew Kay](https://www.mjskay.com/), Northwestern University"
+      - "[Chat Wacharamanotham](https://chatchavan.github.io), Swansea University"
 ---
 
-## What is JoVI?
+# What is JoVI?
```
Comment (mjskay) [inline index.qmd:22]: Why change this to a level-1 heading?

Comment (floe) [reply index.qmd:22]: If you render it as PDF, you get a numbering of 0.1, 0.2 etc. otherwise.

**Comment by mjskay** [comment]
Ah but then it looks wrong on the website (and now I'm realizing the Contact page was wrong too, will submit a PR for that)

**Comment by mjskay** [comment]
Maybe just turn off numbering in the PDF headings? Since it's a one-off anyway

**Pull request merged**